### PR TITLE
541 add missing info column

### LINKED ISSUE #541: Move missing info icon into it's own sortable column
SortedbyMissingInfo

### PR DESCRIPTION
**Pull request by kmagida110**
Fixes #541

### CODE REVIEW

**Review by bamader — APPROVED** [review]
LGTM!

File: client/src/containers/Roster/TableColumns.tsx [hunk 1]
```diff
@@ -18,20 +18,41 @@ export const tableColumns: (_?: boolean) => Column<Child>[] = (
...
-              to={{
-                pathname: `/edit-record/${row.id}`,
-              }}
-            >
-              {row.lastName}, {row.firstName}{' '}
-              {(!!row.validationErrors && row.validationErrors.length == 0) ||
-                row.validationErrors == undefined ||
-                InlineIcon({ icon: 'incomplete' })}
-            </Link>
+            {(!!row.validationErrors && row.validationErrors.length == 0) ||
```
Comment (mmazanec22) [inline client/src/containers/Roster/TableColumns.tsx:53]: !row.validationErrors || !row.validationErrors.length should cover all of this I think?

**Review by mmazanec22 — APPROVED** [review]

File: client/src/containers/Roster/TableColumns.tsx (hunk 1 above)
Comment (jlhogan) [reply client/src/containers/Roster/TableColumns.tsx:53]: agreed, i have this change in a diff branch but simpler conditionals is always a win IMO!!

File: client/src/containers/Roster/TableColumns.tsx [hunk 2]
```diff
@@ -18,20 +18,41 @@ export const tableColumns: (_?: boolean) => Column<Child>[] = (
...
+        ) {
+          return 1;
+        } else {
+          return 0;
+        }
+      },
+
       cell: ({ row }) => {
         return (
           <th scope="row" className={tableRowClassName}>
```
Comment (jlhogan) [inline client/src/containers/Roster/TableColumns.tsx:52]: this column should not be a th, should be td like the other rows

**Review by jlhogan — CHANGES_REQUESTED** [review]

File: client/src/containers/Roster/TableColumns.tsx (hunk 1 above)
Comment (kmagida110) [reply client/src/containers/Roster/TableColumns.tsx:50]: 👍

File: client/src/containers/Roster/TableColumns.tsx (hunk 2 above)
Comment (kmagida110) [reply client/src/containers/Roster/TableColumns.tsx:49]: Sounds good, copied this from the name, should name be td too?

Comment (jlhogan) [reply client/src/containers/Roster/TableColumns.tsx]: nope :) name is the header for the row (thus, `scope="row"`)

Comment (kmagida110) [reply client/src/containers/Roster/TableColumns.tsx]: Fix is pushed, thanks @jlhogan

**Review by jlhogan — APPROVED** [review]

**Pull request merged**